style/sass/scss-lint-config.yml

```diff
@@ -0,0 +1,237 @@
+# Up-to-date with SCSS-Lint v0.43.2
```

Comment:
@gylaz SCSS-Lint is pretty actively developed and [sees a lot of updates](https://github.com/brigade/scss-lint/releases). Hound is a key part of all of this, but I also run SCSS-Lint locally from my command line as I develop to double check style on my current work.

Since a new release from SCSS-Lint is likely to introduce to options and linters, having this comment right in the config file for a project is a nice quick-check to see what version it's for.